src/errors.js

```diff
@@ -536,4 +536,15 @@ Sk.abstr.setUpInheritance("StopIteration", Sk.builtin.StopIteration, Sk.builtin.
 goog.exportSymbol("Sk.builtin.StopIteration", Sk.builtin.StopIteration);
 
 
+// TODO: Extract into sys.exc_info(). Work out how the heck
+// to find out what exceptions are being processed by parent stack frames...
```

Comment:
Yeah lets not worry about this too much, fat chance we'll ever have stack frames. 😄